ntoskrnl/ke/amd64/stubs.c

```diff
@@ -423,4 +423,29 @@ NtVdmControl(IN ULONG ControlCode,
     return STATUS_NOT_IMPLEMENTED;
 }
 
+VOID
+RtlSetUnwindContext(
+    _In_ PCONTEXT Context,
+    _In_ DWORD64 TargetFrame);
+
+VOID
+KiSetTrapContextInternal(
+    _Out_ PKTRAP_FRAME TrapFrame,
+    _In_ PCONTEXT Context,
+    _In_ KPROCESSOR_MODE RequestorMode)
+{
+    ULONG64 TargetFrame;
 
+    /* Save the volatile register context in the trap frame */
+    KeContextToTrapFrame(Context,
+                         NULL,
+                         TrapFrame,
+                         Context->ContextFlags,
+                         RequestorMode);
+
+    /* The target frame is 0x80 bytes before the trap frame */
+    TargetFrame = (ULONG64)TrapFrame - 0x80;
```

Comment:
Isn't there some "sizeof(...)"/"FIELD_OFFSET" or some define to use instead of this hardcoded value?

Reply:
I'll have a look, but this commit breaks compilation anyway, so it will be deferred.